src/cpp/session/include/session/SessionOptions.gen.hpp

```diff
@@ -271,7 +271,7 @@ class GeneratedOptions
       value<bool>(&autoReloadSource_)->default_value(false),
       "Indicates whether or not to automatically reload R source if it changes during the session.")
       ("r-compatible-graphics-engine-version",
-      value<int>(&rCompatibleGraphicsEngineVersion_)->default_value(12),
+      value<int>(&rCompatibleGraphicsEngineVersion_)->default_value(13),
```

Comment:
This should have been changed in the `session-options.json` file and then this `.gen.hpp` file should have been regenerated by running the `generate-options.R` script. I'm making changes in this area now so don't worry about it - I'll bump it to 13.